darwin/future/data_objects/workflow.py

```diff
@@ -0,0 +1,172 @@
+from datetime import datetime
+from enum import Enum
+from typing import List, Optional
+from uuid import UUID
+
+from pydantic import Field
+
+from darwin.future.data_objects.typing import UnknownType
+from darwin.future.pydantic_base import DefaultDarwin
+
+
+class WFDataset(DefaultDarwin):
```

Comment:
`WFDataset` is a simple model, and pretty much exactly what we need it to be already.